File: Modul 2/Bahasa Python/PRAK204-2210817110014-Muhammad Firas.py
angka = [0,1]
angka = list(map(float, input("").split()))

jari2 = float(angka[0])
tinggi = float(angka[1])
phi = 3.142857

volume = phi*jari2*jari2*tinggi
luas = 2*phi*jari2*(jari2+tinggi)
keliling = 2*phi*jari2

print("Volume = %.2f" %volume)
print("Luas = %.2f" %luas)
print("Keliling = %.2f" %keliling)

File: Modul 2/Bahasa C/PRAK204-2210817110014-Muhammad Firas.c
#include<stdio.h>

int main() {
    double phi = 3.142857, r, t;
    scanf("%lf %lf", &r, &t);

    double volume = phi*r*r*t;
    double luas = 2*phi*r*(r+t);
    double keliling = 2*phi*r;

    printf("Volume = %.2lf\n", volume);
    printf("Luas = %.2lf\n", luas);
    printf("Keliling = %.2lf", keliling);


}